_features/fr/multi-language.md

```diff
@@ -1,10 +1,11 @@
 ---
-title: Multi language
+title: Multilingue
 documentation: /documentation/translations.html
 oss: true
 hosted: true
 order: 30
 lang: fr
 ---
 
-{% include features/multi-language.md %}
+Kimai peut être localisé dans n'importe quelle langue et est déjà traduit en 
+**arabe, basque, chinois, croate, tchèque, danois, néerlandais, anglais, espéranto, féroïen, finnois, français, allemand (y compris autrichien et suisse), grec, hébreu, hongrois, italien, japonais, coréen, persan, polonais, portugais, portugais (brésilien), roumain, russe, slovaque, espagnol, suédois, turc et vietnamien**.
```

Comment:
This list looks like it should probably only exist once, otherwise I will forget to adapt it when a new language is added. I guess the best is to keep the start sentence "Kimai peut être localisé dans n'importe quelle langue et est déjà traduit en" in here and use the include afterwards. I will change the other languages the same way